src/test/resources/application.yml

```
@@ -0,0 +1,21 @@
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create-drop # 테스트 종료 후 데이터베이스 정리
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)